adapters/postgres.go

```diff
@@ -235,10 +277,44 @@ func (p *Postgres) patchTableSchemaInTransaction(wrappedTx *Transaction, patchSc
 			return fmt.Errorf("Error patching %s table with '%s' - %s column schema: %v", patchSchema.Name, columnName, mappedColumnType, err)
 		}
 	}
+	// PKFields of the diff set to nil when no modification required. If the value is an empty array, it means that
```

Comment:
I'm not sure if this is the best way to use here. Maybe something like Optional in java should be used here